Package.swift

```diff
@@ -19,7 +19,8 @@ let package = Package(
 	],
 
 	dependencies: [
-        .package(url: "https://github.com/ittybittyapps/PactSwiftMockServer.git", branch: "main")
+        .package(url: "https://github.com/ittybittyapps/PactSwiftMockServer.git", branch: "main"),
+        .package(url: "https://github.com/pointfreeco/swift-snapshot-testing", exact: "1.16.0")
```

Comment:
note: As a general rule in PactSwift I wanted to avoid 3rd party dependencies and stick to what Swift and Xcode provide us. Saying that, I'm a fan of PointFree's mission and what they do myself and pulling in their tools/frameworks that can increase confidence is okay with me. 👍

Reply:
I'm planning to make a couple PRs one with snapshot testing and one without. Like I say, I'm not set on it so I want you to choose what you like.

Reply:
Great stuff. Really appreciate your help! 🙇